Pod/Core/XCTestCase+Gherkin.swift

```diff
@@ -26,6 +26,12 @@ class GherkinState {
     // Used to track step nesting i.e. steps calling out to other steps
     var currentStepDepth: Int = 0
     
+    // file from where currently executed step was invoked
+    var file: StaticString!
```

Comment:
Instead of two properties can we make this a single property i.e.

```swift
/// File and line where the currently executing step was invoked
var currentStepLocation: (file: StaticString, line: Uint)?
```

Doesn't make much difference to the implementation, but it does convey the intent of the property more clearly. It also makes sure that we don't accidentally set one without the other being set too :)